js/modules/Debug/Debug.js

```diff
@@ -576,14 +584,15 @@ window.GLPI.Debug = new class Debug {
                 }
             }
 
-            const editor = window.CodeMirror(container.get(0), {
-                value: data_string,
-                mode: 'application/json',
-                lineNumbers: true,
-                readOnly: true,
-                foldGutter: true,
-                gutters: ['CodeMirror-linenumbers', 'CodeMirror-foldgutter'],
+            const editor = new window.CodeMirror.EditorView({
+                extensions: [
+                    window.CodeMirror.setup,
+                    window.CodeMirror.languages.json(),
```

Comment:
This should make it readonly.
```suggestion
                    window.CodeMirror.languages.json(),
                    window.CodeMirror.EditorView.contentAttributes.of({contenteditable: false}),
```